releasenotes/notes/expose-is_onehot-044d860d5eb7b115.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Expose method ``is_onehot``.
```

```suggestion
    Add ``Constraint.is_onehot()`` method.
```
We know that we're just exposing it, but not sure a typical user does.